format/index.md

```diff
@@ -300,6 +300,11 @@ pagination links, as described below.
 If a link object refers to resource objects included in the same compound
 document, it **MUST** include resource linkage to those resource objects.
 
+> Note: a resource may actually be referring to another resource returned 
+in the same compound document, but this reference may be omitted 
+from the link object of the parent resource because of sparse fieldset 
+restrictions.
+
```

Comment:
I think the exception on **MUST** must be normative.

Instead of:

> If a link object refers to resource objects included in the same compound document, it **MUST** include resource linkage to those resource objects.

Let's consider:

> If a link object refers to resource objects included in the same compound document, it **MUST** include resource linkage to those resource objects unless those fields are excluded via sparse fieldset restrictions.

The non-normative note could then be removed.